blueocean-dashboard/src/main/js/util/UrlUtils.js

```diff
@@ -19,6 +19,14 @@ export const buildPipelineUrl = (organization, fullName, tabName) => {
     return tabName ? `${baseUrl}/${tabName}` : baseUrl;
 };
 
+export const buildClassicConfigUrl = (window, pipeline) => {
+    if (pipeline && pipeline.fullDisplayName) {
+        return `${window.jenkinsCIGlobal.rootURL}/job/${pipeline.fullDisplayName.split('/').join('/job/')}/configure`;
```

Comment:
Prob best not to access this via `window.jenkinsCIGlobal.rootURL` if it can be avoided. I think you should be able to use `getJenkinsRootURL` in the `config.js` module.